### PR TITLE
[JS] Topic: Web Browser APIs

### LINKED ISSUE #2020: [JS] Topic: Web Browser APIs
### Description

# Description
- What are APIs?
- What can APIs do?
- How do APIs work?
<br>

# Directory
```Topics/14.Web_Browser_APIs```

### Domain

JavaScript

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by sushovanb02**
<hr>

## Description 📜

Created a document on the Introduction to Web Browser APIs.
Fixes #2020 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] JavaScript

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by sushovanb02** [comment]
@paramsiddharth @s4-2 @Creatoon please review this PR on Introduction to Web APIs

**Comment by paramsiddharth** [comment]
@Creatoon @dakshsethi @s4-2 Please review.

File: JavaScript/Topics/14.Web_Browser_APIs/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# What is an API?
+ API stands for application programming interface. APIs are the little pieces of code that make it possible for digital devices, software applications, and data servers to talk with each other, and they’re the essential backbone of so many services we now rely on.
```
Comment (s4-2) [inline JavaScript/Topics/14.Web_Browser_APIs/README.md:2]: @sushovanb02 Write API full form using bold mark:)

File: JavaScript/Topics/14.Web_Browser_APIs/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+# How do APIs work?
+APIs work by sharing data and information between applications, systems, and devices—making it possible for these things to talk with each other.
+
+Sometimes the easiest way to think about APIs is to think about a metaphor, and a common scenario that a lot of folks use is that of the customer, a waiter, and a restaurant kitchen: A customer talks to the waiter and tells the waiter what she wants. The waiter takes down the order and communicates it to the kitchen. The kitchen does their work, creating the food, and then the waiter delivers the order back to the customer.
+
+In this metaphor, a customer is like a user, who tells the waiter what she wants. The waiter is like an API, receiving the customer’s order and translating the order into easy-to-follow instructions that the kitchen then uses to fulfill that order—often following a specific set of codes, or input, that the kitchen easily recognizes. The kitchen is like a server that does the work of creating the order in the manner the customer wants it, hopefully! When the food is ready, the waiter picks up the order and delivers it to the customer.
+
+#  Types of APIs
+- ## Browser APIs
```
Comment (s4-2) [inline JavaScript/Topics/14.Web_Browser_APIs/README.md:37]: Please provide number in types of API. Eg 1)- Browser API

File: JavaScript/Topics/14.Web_Browser_APIs/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  }
+}
+
+function showPosition(position) {
+  myElement.innerHTML = "Latitude: " + position.coords.latitude +
+  "<br>Longitude: " + position.coords.longitude;
+}
+```
+
+- ## Third Party APIs
```
Comment (s4-2) [inline JavaScript/Topics/14.Web_Browser_APIs/README.md:60]: @sushovanb02 Describe more about Third Party API. Hope you got it:)

Comment (sushovanb02) [reply JavaScript/Topics/14.Web_Browser_APIs/README.md:60]: yup!

**Comment by sushovanb02** [comment]
@s4-2 done all the changes that you suggested. Please check

**Review by dakshsethi — APPROVED** [review]
Looks good to me!! 🔥🔥🔥

**Review by s4-2 — APPROVED** [review]
@sushovanb02 Good job, looks good to me.
Keep up it:)

**Comment by sushovanb02** [comment]
@paramsiddharth 2 mentors have approved now you can merge ig

**Review by Creatoon — APPROVED** [review]

**Review by paramsiddharth — APPROVED** [review]

**Pull request merged**